deployer/src/persistence/resource/database.rs

```diff
@@ -88,6 +88,7 @@ impl Display for Type {
     }
 }
 
+// TODO: Unify with shuttle_common::resource::Type
 impl FromStr for Type {
```

Comment:
I think this `FromStr` can now be removed right?

Reply:
Indeed